src/math/Interpolant.js

```diff
@@ -47,7 +47,7 @@ Interpolant.prototype = {
 
 		validate_interval: {
 
-			seek: {
+			seek: while(true) {
```

Comment:
Yeah it's hacky however it doesn't affect performance at all. due to the fact that the while always breaks. It's more of a temp fix.